### ISSUE TITLE
Publication listed twice for "Akt1 Has Increased Activity Or Abundance Caused By Semaglutide"

### ISSUE THREAD

**Issue by khanspers** [NCATSTranslator/Feedback#968]
Query: What genes' activity may be increased by [Semaglutide](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:167574)?
https://ui.test.transltr.io/results?l=Semaglutide&i=CHEBI:167574&t=3&r=0&q=e09c360d-dab0-4346-997f-3575ec243b27

AKT1 and AKT2 are both returned with direct paths, but the one publication listed as evidence (same for both) is listed twice:

<img width="1216" alt="Screen Shot 2024-10-29 at 12 33 04 PM" src="https://github.com/user-attachments/assets/4b3fc65b-417d-4e41-8996-5ee87374a48b">
<img width="1218" alt="Screen Shot 2024-10-29 at 12 32 54 PM" src="https://github.com/user-attachments/assets/5185496e-f858-4944-93e8-a9ab9f5253cd">

Knowledge Source is Text Mining Targeted Association API.

**Comment by gaurav** [comment]
I don't think this will help you, but NodeNorm can normalize PMC IDs to PMIDs for any articles published whenever we last built Babel ([example](https://nodenorm.transltr.io/1.5/get_normalized_nodes?curie=PMC%3APMC3406479&conflate=true&drug_chemical_conflate=false&description=false&individual_types=false)).

(I don't think this will help because it's probably easier to call [NCBI's eutils service](https://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pmc&db=pubmed&id=3406479), which should be up-to-date to the present date and is only a little harder to parse than NodeNorm's output, as its XML instead of JSON.)
